pom.xml

```diff
@@ -181,6 +181,11 @@
         <activeMQVersion>5.8.0</activeMQVersion>
         <commonsLangVersion>2.6</commonsLangVersion>
 
+        <!-- Elasticsearch -->
+        <elasticsearchMavenPluginVersion>2.0</elasticsearchMavenPluginVersion>
+        <!-- Keep in sync with the dependency required by Elasticsearch -->
+        <elasticsearchGroovyVersion>2.4.4</elasticsearchGroovyVersion>
```

Comment:
@Sanne I added the versions and the dependencies in the root pom.xml but you did it in the elasticsearch/pom.xml file for the Elasticsearch test dependency. We should be consistent: what do we prefer? Personnally I like to have all the versions in the root pom.xml so that I don't forget to update a dependency and it looks consistent with the fact that arquillian versions are in the root pom.xml.

Thanks for the feedback.